aws/ecs/exec_command.go

```diff
@@ -13,6 +13,9 @@ func ExecCommand(ctx context.Context, infra Outputs, taskId string, containerNam
 	if containerName == "" {
 		containerName = infra.MainContainerName
 	}
+	if len(cmd) == 0 {
```

Comment:
I don't think we want to fill a command here.

Reply:
For an ECS task, I would agree. But I think this can also be used for an ECS Service. I think this code is carry over from what was already here. In the case of an ECS Service, what do you think should happen?

Reply:
I think it would be better to backfill in the `ecs.Remoter`. That way, you can backfill only in the case of a service.

Reply:
Oh, never mind, I missed something. It looks like this code only applied to services and there is a separate function for running tasks. Ignore this recommendation.